reasoner/ReasonerProducer.java

```diff
@@ -47,8 +49,7 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ReasonerProducer.class);
 
-    private static final int COMPUTE_SIZE = Executors.PARALLELISATION_FACTOR * 2;
-
+    private int computeSize = Executors.PARALLELISATION_FACTOR * 2;
```

Comment:
we normally only initialise static members like this - everything else should be in the constructor